tests/Microsoft.Identity.Test.Unit/BrokerTests/RuntimeBrokerTests.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-#if !NET48 && !NET7_0
+#if !NET6_0
+#if !NET48
```

Comment:
if NET6_WIN ?

Reply:
Has to be net6, Broker stuff seems to not be available on Net6